### PR TITLE
bazel/linux: Update README.md

### PR DESCRIPTION
**Pull request by stefanoen**
Update the linux kernel bazel rules README to document what has changed with the new version of the kernel_test rule.

### CODE REVIEW

File: bazel/linux/README.md [hunk 1]
```diff
@@ -2,33 +2,41 @@
 Bazel hooks for compiling and testing linux kernel modules against a specific kernel version.
 
 # Main rules made available
-`kernel_version` defines a specific kernel version to compile and test modules against.
+`kernel_tree_version`* defines a specific kernel tree to compile and test modules against.
```
Comment (ccontavalli) [inline bazel/linux/README.md:5]: What is the '*' after kernel_tree_version? not sure what it achieves. Italic? but there is no closing *. Bold? same.

File: bazel/linux/README.md [hunk 2]
```diff
@@ -2,33 +2,41 @@
...
+1. Declare how to build the test kernel module following the instructions above
+2. Add a `rootfs_version(name="my-rootfs", ulr="$URL", ...)` rule in your repository WORKSPACE file
+3. Add a `kernel_image_version(name="my-kernel-image", ulr="$URL", ...)` rule in your repository WORKSPACE file
    * Check out the *generate_custom_archive.sh* [instructions](https://github.com/enfabrica/enkit/blob/master/kbuild/utils/README.md) if you don't know how to generate a suitable  user-mode linux executable image
-4. Add a `kernel_test(name="my-kernel-module-test", module=":my-kernel-module", kernel_image="@kernel-img//file", rootfs_image="@rootfs-img//file")` rule inside your BUILD.bazel file in your kernel module directory
-5. Test the kernel running `bazel test :my-kernel-module-test`
+4. Add a `kernel_test(name="my_module_test", module=":my_module", kernel_image="@my-kernel-image", rootfs_image="@my-rootfs")` rule inside your BUILD.bazel file in your kernel module directory
+5. Test the kernel running `bazel test :my_module_test`
+   * Use `--test_output=all` if you want to see the user-mode linux and kunit output
```
Comment (ccontavalli) [inline bazel/linux/README.md:36]: This is all very good documentation! As a side note, I think we should have an example checked in in the public repo.

**Review by ccontavalli — APPROVED** [review]

File: bazel/linux/README.md (hunk 1 above)
Comment (stefanoen) [reply bazel/linux/README.md:5]: My intention was to use it as as a reference to the footnote (line 20). Does markdown have "native" footnotes?

File: bazel/linux/README.md (hunk 2 above)
Comment (stefanoen) [reply bazel/linux/README.md:36]: Agreed. I'll come up with an E2E example for running a test.

**Pull request merged**